elide-contrib/elide-test-helpers/src/main/java/com/yahoo/elide/contrib/testhelpers/jsonapi/JsonApiDSL.java

```diff
@@ -231,6 +231,28 @@ public static Relation relation(String field, ResourceLinkage... resourceLinkage
         return new Relation(field, resourceLinkage);
     }
 
+    /**
+     * Creates a to-one relationship
+     *
+     * @param field           the field
+     * @param resourceLinkage the resource linkage
+     * @return the relation
+     */
+    public static Relation toOneRelation(String field, ResourceLinkage... resourceLinkage) {
```

Comment:
I had considered adding functions like this but opted instead to add the TO_ONE, TO_MANY params.  I think we should get a second opinion on which way to go.  I prefer the params and no new functions.  @jkusa ?